SilasKenneth/fast-food-api-v2
# [Finishes #160915103] Create the auth endpoints and tests

### What this pull request does?
- The pull request contains the endpoints for the user signup and login
- It contains the tests_for both the login and the signup
### How to test
 - To test the project you should have atleast python version `3.5`
 - Clone the repo
 - cd to the project directory
 - create a virtual environment through `virtualenv -p python3` 
- Activate the virtualenv through `source env/bin/activate`
- Install packages through `pip install -r requirements.txt`
- Run `pytest`
